File: server/src/main/java/pt/iade/ei/thinktoilet/models/dtos/RegisterRequest.java
package pt.iade.ei.thinktoilet.models.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RegisterRequest {
    private String name;
    private String email;
    private String password;
    private String iconId;
    private LocalDate birthDate;
}

File: app/app/src/main/java/pt/iade/ei/thinktoilet/models/RegisterRequest.kt
package pt.iade.ei.thinktoilet.models

import com.google.gson.annotations.SerializedName
import java.time.LocalDate

data class RegisterRequest(
    @SerializedName("name") val name: String,
    @SerializedName("email") val email: String,
    @SerializedName("password") val password: String,
    @SerializedName("iconId") val iconId: String?,
    @SerializedName("birthDate") val birthDate: String?
)
